edge-modules/TestAnalyzer/Controllers/DirectMethodStatusController.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer.Controllers
+{
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc;
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class DirectMethodStatusController : Controller
+    {
+        // POST api/directmethodstatus
+        [HttpPost]
+        public async Task<ActionResult<bool>> Post(ResponseStatus methodCallStatus)
+        {
+            await ReportingCacheWithStorage.Instance.AddDirectMethod(methodCallStatus);
+            return true;
```

Comment:
why need to always return true here?

Reply:
The return type is ActionResult<bool>. I tried to only use ActionResult without type but the build complained about not being able to implicitly convert to bool.

Reply:
try to return Task<EmptyResult>